acme/acme/challenges.py

```diff
@@ -138,76 +200,67 @@ def simple_verify(self, chall, domain, port=None):
                 "Using non-standard port for SimpleHTTP verification: %s", port)
```

Comment:
This code doesn't change in the current PR, but I'm wondering what kind of coordination is happening in the ACME spec to determine whether or not it will be legal to specify port. If it's not allowed there, it might be a good idea to take it out entirely here, and the converse.